### DISCUSSION TITLE
Nominate @TisVictress as a Web Servers team contributor

### DISCUSSION THREAD

**Discussion by TisVictress** [paketo-buildpacks/feedback#174] [Web Servers Team]
Hey! I'd like to nominate myself as a contributor for web servers. I've recently done a lot of work on web servers via the following commits:

https://github.com/paketo-community/web-servers/commit/38c3d1b43ac1a4fa7850542c531fe078972f9365
https://github.com/paketo-buildpacks/httpd/commit/c0e1e2d668e3f12aacf8a6b43f713ac93d19db93
https://github.com/paketo-buildpacks/nginx/commit/f6feeb3233282fe746bc2e955e7b5f840b034255

@paketo-buildpacks/web-servers-maintainers 👍 / 👎 ?

**Comment by ryanmoran** [comment]
Congrats on becoming a Web Servers contributor, @TisVictress! 🎉
